lib/hooks/pre/downcloud.js

```diff
@@ -0,0 +1,110 @@
+#!/usr/bin/env node
+
+import fs from 'fs-extra'
+import tar from 'tar'
+import { spawn } from 'child_process'
+
+const BUILD_FOLDER = './build/'
+const COZY_URL = 'downcloud.cozycloud.cc'
+
+const launchCmd = (cmd, params, options) => {
```

Comment:
`spawnPromise` would be clearer IMHO.

Reply:
Meh, can't say I agree tbh. `spawn` is very node-oriented, but I think `launchCmd ` can be understood by everyone. And I don't think the function name should reflect that it returns a promise.